.claude/hooks/utils/constants.py

```
@@ -0,0 +1,108 @@
+                    break
+                except OSError as e:
+                    if attempt == max_retries - 1:
+                        raise e
+                    # Brief random delay to reduce collision probability
+                    time.sleep(random.uniform(0.01, 0.05))
+        finally:
+            # Clean up lock file if it exists
+            try:
+                lock_file.unlink(missing_ok=True)
+            except OSError:
+                pass  # Lock file cleanup is best-effort
```

Ruff (0.12.2)

Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

(SIM105)